README.md

```diff
@@ -6,10 +6,10 @@ so this really strange thing happened
 i started poking around on github
 not really knowing what i was doing
 I'm sure other products do some things differently.
-But is the plumbing all the same?
+taking this away.
```

Comment:
okso when there is a diff you can comment on each. when there isn't, then you comment on both. ok.